test/cypress/support/output-logs.js

```diff
@@ -0,0 +1,50 @@
+// ***********
+// Output App logs in case a test fails!
+// Copied directly from: https://github.com/cypress-io/cypress/issues/3199#issuecomment-466593084
+// ***********
```

Comment:
@taoeffect This is the solution I found to make cypress output the app logs in case a test spec fails.
I added a commit to force the fail, it was this run: https://dashboard.cypress.io/projects/q6whky/runs/318/specs (click on output icon next to the first spec failing.

I noticed sometimes it says `[object, object]`. I tried to use `JSON.stringify()` as an attempt to show the original value but it didn't work. Do you have any other idea?

Reply:
Hmm, did you try not doing `args.join(' ')` before passing the args to stringify? e.g. just `JSON.stringify(args)`?

Reply:
Actually it was easier than that. Just passing `args` with using template literals \` \` worked perfectly :D.